### ISSUE TITLE
npm install of chips not working

### ISSUE THREAD

**Issue by AlexMontgomerie** [material-components/material-components-web-react#311]
Not sure what the issue is, but I keep getting the following error from importing @material/react-chips (import {ChipSet, Chip} from '@material/react-chips';). 

> ERROR in ./node_modules/@material/react-chips/ChipCheckmark.js
    Module parse failed: Unexpected token (26:8)
    You may need an appropriate loader to handle this file type.
    | 
    | export default class ChipCheckmark extends Component {
    |   width = null;
    | 
    |   init = (element) => {
     @ ./node_modules/@material/react-chips/index.js 24:0-44
     @ ./components/ProfileEdit.js
     @ ./components/App.js
     @ ./index.js
     @ multi (webpack)-dev-server/client?http://localhost:8085 ./index.js
    
> ERROR in ./node_modules/@material/react-chips/Chip.js
    Module parse failed: Unexpected token (30:15)
    You may need an appropriate loader to handle this file type.
    | 
    | export class Chip extends Component {
    |   chipElement_ = null;
    |   foundation_ = null;
    |   state = {
     @ ./node_modules/@material/react-chips/index.js 23:0-26
     @ ./components/ProfileEdit.js
     @ ./components/App.js
     @ ./index.js
     @ multi (webpack)-dev-server/client?http://localhost:8085 ./index.js
    
   > ERROR in ./node_modules/@material/react-chips/ChipSet.js
    Module parse failed: Unexpected token (31:18)
    You may need an appropriate loader to handle this file type.
    | 
    | export default class ChipSet extends Component {
    |   checkmarkWidth_ = 0;
    |   foundation_ = null;
    |   state = {
     @ ./node_modules/@material/react-chips/index.js 25:0-32
     @ ./components/ProfileEdit.js
     @ ./components/App.js
     @ ./index.js
     @ multi (webpack)-dev-server/client?http://localhost:8085 ./index.js

my webpack.config.js and package.json are as follows

```
module.exports = [
  {
    entry: "./index.js",
    output: {
      filename: "bundle.js"
    },
    module: {
      rules: [{
        test: /\.scss$/,
        use: [
          {
            loader: 'file-loader',
            options: {
              name: 'bundle.css',
            },
          },
          { loader: 'extract-loader' },
          { loader: 'css-loader' },
          {
            loader: 'sass-loader',
            options: {
              includePaths: ['./node_modules'],
            }
          },
        ]
      }, {
        test: /\.js$/,
        exclude: /(node_modules)/,
        loader: 'babel-loader',
      },]
    },
  },
];
```

```
{
  "name": "test",
  "private": true,
  "version": "0.0.0",
  "scripts": {
    "start": "webpack-dev-server --progress",
    "deploy": "webpack --config webpack.config.js --progress"
  },
  "license": "Apache-2.0",
  "devDependencies": {
    "babel-core": "^6.26.3",
    "babel-loader": "^7.1.5",
    "babel-plugin-transform-class-properties": "^6.24.1",
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "css-loader": "^0.28.0",
    "extract-loader": "^1.0.2",
    "file-loader": "^1.1.11",
    "node-sass": "^4.9.0",
    "sass-loader": "^6.0.4",
    "webpack": "^3.12.0",
    "webpack-dev-server": "^2.11.3"
  },
  "dependencies": {
    "@material/react-chips": "^0.5.2",
  }
}
```
I noticed that after i installed with npm and ran npm start, the node_modules folder contained un-compiled(?) React code for chips. I don't know if this is an issue with my webpack setup. All other node module folders have a dist folder with non-react js in it. Has anyone encountered this issue? 

Also would like to say it's great work you guys are doing, it's been very useful.

Thanks,
Alex

**Comment by moog16** [comment]
@AlexMontgomerie thanks for the kind words...glad the components are useful.

What version of Chips are you on? It looks like you have 0.5.2 in your package.json, but maybe you have 0.5.1? There was an issue with installing 0.5.1.

Another debug tip: If you look in the `node_modules/@material/react-chips/package.json` file, do you see a 'main' prop? It should be `main: /dist/index.js`.

Also you could try deleting your node_modules folder `rm -rf node_modules`, then running `npm install` again. Worth a shot...

If all else fails, please shoot us a copy of your project (or subset) using Chips. Just zip it up and send it over in this issue (without node_modules please).

**Comment by AlexMontgomerie** [comment]
[chips_issue.zip](https://github.com/material-components/material-components-web-react/files/2443579/chips_issue.zip)

@moog16 Thanks for replying. Here's a subset of my project. Steps to recreate are `npm i && npm start`. The react-chips component is imported in the components/ProfileEdit.js file. It seems fine importing the other material react components, which is the only reason i brought this up. I've noticed that in my package-lock.json, it references this link for react-chips: https://registry.npmjs.org/@material/react-chips/-/react-chips-0.5.2.tgz. And when I download that, there is no dist folder.I believe I'm downloading 0.5.2 as that's what it suggests in both the package.json and package-lock.json. Could it possibly be an issue with my webpack config? It seems that the react-chips module has a different structure to others, which might not work with my webpack setup. 

Thanks,
Alex

**Comment by moog16** [comment]
Wow you're right. I think when I tested on my local, I had a previous version. Will fix and do another patch release.

**Comment by moog16** [comment]
released 0.5.3!

**Comment by AlexMontgomerie** [comment]
cheers for that!
